rust/stackablectl/src/cmds/operator.rs

```diff
@@ -86,6 +89,10 @@ Use \"stackablectl operator list\" to list available versions for all operators
 Use \"stackablectl operator describe <OPERATOR>\" to get available versions for one operator")]
     operators: Vec<OperatorSpec>,
 
+    /// Namespace in the cluster used to deploy the operators
+    #[arg(short = 'n', long, default_value = DEFAULT_OPERATOR_NAMESPACE)]
```

Comment:
Here the -n makes sense to point to operators i think :+1:

Reply:
Removed in 7066caa